sample/host.json

```diff
@@ -4,7 +4,11 @@
     "sendGrid": {
         "from": "Azure Functions <samples@functions.com>"
     },
+    "http": {
```

Comment:
If customers want to remove the "api" prefix completely, they can set this to empty here